pkg/api/accesscontrol.go

```diff
@@ -549,7 +549,7 @@ func (hs *HTTPServer) declareFixedRoles() error {
 	libraryPanelsGeneralWriterRole := ac.RoleRegistration{
 		Role: ac.RoleDTO{
 			Name:        "fixed:library.panels:general.writer",
-			DisplayName: "Root level writer",
```

Comment:
@eleijonmarck RE the general folder comment thread, should this be changed from root to general as well? I'm generally in favor of some sort of ubiquitous language when referring to the same thing but don't want to go too crazy renaming things if it doesn't make sense.